src/doc/DocAnnotator.js

```diff
@@ -632,8 +629,14 @@ class DocAnnotator extends Annotator {
      */
     highlightMousedownHandler = (event) => {
         this.isCreatingHighlight = true;
-        this.mouseX = event.clientX;
-        this.mouseY = event.clientY;
+
+        let mouseEvent = event;
```

Comment:
Extract the mouse position calculation to a function that returns and/or sets `this.mouseX/Y`